src/components/10-atoms/fieldset/demo.js

```diff
@@ -0,0 +1,53 @@
+/* global document */
+import { storiesOf } from '@storybook/html';
+import { html, render } from 'lit-html';
+import './index';
+
+import Readme from './README.md';
+import Changelog from './CHANGELOG.md';
+
+storiesOf('Components|Atoms/Fieldset/Demos', module)
+  .addParameters({
+    readme: {
+      sidebar: Readme,
+    },
+    changelog: Changelog,
+  })
+  .add('Fieldset - enableresponsivestretch', () => {
+    const root = document.createElement('div');
+
+    const template = html`
+      <axa-text variant="size-3">
+        Resize your browser to a tablet or mobile width to see the effect of
+        "enableResponsiveStretch"</axa-text
+      >
+      <br />
+      <axa-text>How old are you?</axa-text>
+      <br />
+      <axa-fieldset horizontal enableresponsivestretch>
```

Comment:
This line shows that `enableresponsivestretch` has redundant information and hence should be shortened. It's a Boolean attribute, so the presence automatically 'enables' things. Delete the 'enable' bit therefore. Naming matters...

Reply:
Forget the previous comment if you accept the change of `horizontal` data type to string from above...

Reply:
Done